src/components/tasks/card/index.tsx

```diff
@@ -39,33 +39,36 @@ const Card: FC<Props> = ({
   const [showEditButton, setShowEditButton] = useState(false);
   const [keyLongPressed] = useKeyLongPressed();
   const [isLoading, setIsLoading] = useState<boolean>(false);
+
+  const context = useAppContext();
+  const { actions, state } = context || {};
```

Comment:
The term `state` is a fairly general. We could have been more specific like `idleUserState` or a related term.

Reply:
This state here refers to all the states in App context

Reply:
What @heyrandhir might be trying to tell here (and something I agree with) is one context should store and mutate only a single type of data. 
Example: 
- userSettings
- cartItems

And hence it should be named specifically to its use case and not a generic term.

Reply:
Because it is a global state and we are checking for login status I don't think there is a need for a separate one, if it is required then, it would rather be a better idea to call the Appwrapper context, editContext. I can create a refactor ticket for it